src/com/company/Main.java

```diff
@@ -3,37 +3,31 @@
 import java.util.Scanner;
 
 public class Main {
-    static String scann () {
+    static String  scanLine() {
         Scanner scan = new Scanner(System.in);
-           return scan.nextLine();
+        return scan.nextLine();
+    }
+    static long power(int base, int exp){
+        long pow = 1;
+        if (exp == 0) return 1;
```

Comment:
E bine sa folosesti mereu paranteze, chiar si cand e vorba de one-liner-e. E good practice, mai ales ca in cazul aplicatiilor complexe vorbim despre posibilitatea adaugarii/schimbarii/stergerii de comenzi din blocul respectiv, aka punem {} preventiv.